src/sidebar/util/sentry.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const Sentry = require('@sentry/browser');
+
+const warnOnce = require('../../shared/warn-once');
+
+/**
+ * @typedef SentryConfig
+ * @prop {string} dsn
+ * @prop {string} environment
+ */
+
+let eventsSent = 0;
+const maxEventsToSendPerSession = 5;
```

Comment:
How did you decide on 5?

Reply:
I just picked a number that seemed reasonable to me for one session of the client. It means that if multiple errors happen, we'll find out about them, but if there are a cacophony of errors, only the first few will hit Sentry.

My expectation is that we'll adjust the filtering based on what we see happening in production.